skyportal/handlers/api/instrument.py

```diff
@@ -49,6 +49,15 @@ def post(self):
                 'Invalid/missing parameters: ' f'{exc.normalized_messages()}'
             )
 
+        sensitivity_data = data.pop("sensitivity_data", None)
+        if sensitivity_data:
+            filters = data.get("filters")
```

Comment:
Will want to make sure filters is always a list
```suggestion
            filters = data.get("filters", [])
```